### ISSUE TITLE
[1.34.x] fromUnit toUnit isn't saved to mapping.xml when using ConvertVolumeUnits

### ISSUE THREAD

**Issue by mmelko** [atlasmap/atlasmap#433]
<img width="673" alt="screen shot 2018-05-16 at 13 17 41" src="https://user-images.githubusercontent.com/6814482/40114057-c469cba8-590b-11e8-8593-ef4e30f2c97b.png">

```xml
<Mappings>
        <Mapping xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="Mapping" id="mapping.10" mappingType="Map">
            <InputField xsi:type="ns2:JavaField" name="sourceDouble" docId="io.atlasmap.qe.test.SourceMappingTestClass" path="/sourceDouble" fieldType="Double"/>
            <OutputField xsi:type="ns2:JavaField" name="targetDouble" docId="io.atlasmap.qe.test.TargetMappingTestClass" path="/targetDouble" fieldType="Double">
                <Actions>
                    <ConvertVolumeUnit/>
                </Actions>
            </OutputField>
        </Mapping>
    </Mappings>
```

**Comment by igarashitm** [comment]
@mmelko is it possible to try with master? if it works with master then it'll be no problem for 7.1. 1.34 is going to be a hot fix branch for 7.0.

**Comment by igarashitm** [comment]
cf. #401
